conf.yaml

```diff
@@ -1436,35 +1436,50 @@ contents:
     -
         title:      Enterprise Search
         sections:
+          -
+            title:      Enterprise Search Guide
+            prefix:     en/enterprise-search
+            index:      enterprise-search-docs/index.asciidoc
+            private:    1
+            current:    *stackcurrent
+            branches:   [ 7.7 ]
+            live:       *stacklive
+            chunk:      1
+            tags:       Enterprise Search/Guide
+            subject:    Enterprise Search
+            sources:
+              -
+                repo:   enterprise-search-pubs
+                path:   enterprise-search-docs
           -
             title:      Workplace Search Guide
             prefix:     en/workplace-search
             index:      workplace-search-docs/index.asciidoc
             private:    1
-            current:    7.6
-            branches:   [ 7.6 ]
+            current:    *stackcurrent
+            branches:   [ 7.7, 7.6 ]
```

Comment:
@goodroot @jonasll 7.8 branches were added to the other Stack docbooks with #1831.
Let me know if `7.8` should be added here.

Reply:
Thanks @jrodewig !

@chriscressman for visibility. He is back in action. (Enterprise Search doc contact)

Reply:
Yes, please add 7.8. I created that branch in our repo. Thanks @jrodewig !

Reply:
Added with 7e38646.